src/components/accession2/WithdrawModal.tsx

```diff
@@ -267,6 +273,17 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
             onChange={onChangeDate}
           />
         </Grid>
+        <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
+          {isNotesOpened ? (
+            <Textfield id='notes' value={record.notes} onChange={onChange} type='textarea' label={strings.NOTES} />
```

Comment:
oh no, these notes are used for the plant description. yikes, we need to ask Steve maybe.

Reply:
No, no. My record here is a withdrawal, so this field would be accession.withdrawal[0].notes

Reply:
ugh, my bad.. thanks for clarifying :)